tests/cnf/core/network/metallb/tests/bgp-multiservice.go

```diff
@@ -0,0 +1,153 @@
+package tests
+
+import (
+	"fmt"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/configmap"
+	"github.com/openshift-kni/eco-goinfra/pkg/metallb"
+	"github.com/openshift-kni/eco-goinfra/pkg/nad"
+	"github.com/openshift-kni/eco-goinfra/pkg/namespace"
+	"github.com/openshift-kni/eco-goinfra/pkg/nodes"
+	"github.com/openshift-kni/eco-goinfra/pkg/pod"
+	"github.com/openshift-kni/eco-goinfra/pkg/reportxml"
+	"github.com/openshift-kni/eco-goinfra/pkg/service"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/core/network/internal/cmd"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/core/network/internal/frrconfig"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/core/network/internal/ipaddr"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/core/network/internal/netparam"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/core/network/metallb/internal/metallbenv"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/core/network/metallb/internal/tsparams"
+	"k8s.io/apimachinery/pkg/labels"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/openshift-kni/eco-gotests/tests/cnf/core/network/internal/netinittools"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("MetalLB BGP", Ordered, Label(tsparams.LabelBGPTestCases), ContinueOnFailure, func() {
+	var (
+		err           error
+		AddressPoolS1 = []string{"4.4.4.100", "4.4.4.101"}
```

Comment:
Hi Evgeny do we want to try and use DefineIterationParams for the ipaddresspools?